vagrant-k8s-manual/Vagrantfile

```diff
@@ -1,62 +1,92 @@
 $VirtualSwitchName = "External"
 
-LINUX_IMAGE = "generic/ubuntu1604"
+$LinuxImage = "ubuntu/xenial64"
+
+$WindowsImage = "WindowsServer1709Docker"
+$SyncedFolderType = "smb"
 
-IMAGE = "WindowsServer1709Docker"
 ClusterCIDR = "10.4.0.0/16"
 ClusterCIDRShort = "10.4"
 
 Vagrant.configure("2") do |config|
+
+    config.vm.provider "virtualbox" do |v|
+        $WindowsImage = "StefanScherer/windows_2016_docker"
+        $SyncedFolderType = "virtualbox"
```

Comment:
I think that you could reuse the same pattern with a default and virtualbox override for the Linux Image, leaving the generic 16.04 above, and putting `$LinuxImage = "ubuntu/xenial64"` here. Then that doesn't break the Hyper-V support due to no image.